powermock-reflect/src/test/java/org/powermock/reflect/internal/WhiteboxImplTest.java

```diff
@@ -16,16 +16,15 @@
 package org.powermock.reflect.internal;
 
 import org.junit.Test;
-import org.powermock.reflect.testclasses.Child;
-import org.powermock.reflect.testclasses.ClassWithOverloadedMethods;
-import org.powermock.reflect.testclasses.ClassWithStandardMethod;
+import org.powermock.reflect.testclasses.*;
```

Comment:
Please use PowerMock import approach when full class name with pbacked is specified for all classes.